frontend/src/app/containers/ExperimentPage/DetailsDescription.js

```diff
@@ -67,7 +67,7 @@ export default function DetailsDescription({
               </div>}
             <LocalizedHtml id={l10nId("introduction")}>
               <div>
-                {parser(introduction)}
+                {introduction}
```

Comment:
IIUC, parsing the introduction text here uses up CPU cycles just for the localization to take over milliseconds later. This content will only be visible if the `introduction` id is missing from all locale, including English. I think this is unlikely enough to recognize this as a rare error scenario in which displaying unparsed HTML is acceptable.